README.md

```diff
@@ -21,6 +21,7 @@ and push your operator image to quay.io to make it available for installation.
 8. Expose the new IQ Server outside the cluster: 
    2. Create a Route in OpenShift UI to the new service, port 8070.
 9. Visit the new URL shown on the Route page in OpenShift UI.
+10. Default credentials are `admin`/`admin123`.
```

Comment:
Should we include a note here to remind people to update these values?

Reply:
upon login, it'll remind you, but i can mention it here as well, in case they don't recognize the impact of the default logins.

Reply:
This is a doc for dev/testing, so I've not bothered mentioning it here, but I have added a note about setting credentials over in the description/docs that are shown in the operator hub that users would see.